apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -108,6 +112,10 @@ export interface GeneralDetailsFormValuesInterface {
      * Is the application discoverable.
      */
     discoverableByEndUsers?: boolean;
+    /**
+     * Is SaaS application.
+     */
+    saas?: boolean;
```

Comment:
Isn't this serve the same purpose as `isSaasApp` prop?

Reply:
Nope. `saas` is a member of the form values interface which is used to send the updated saas configuration. `isSaasApp` is a prop that indicates the current saas config value which is used as the initial value for the `saas` value in L177.